pkg/core/blockchain.go

```diff
@@ -537,6 +543,14 @@ func (bc *Blockchain) jumpToStateInternal(p uint32, stage stateJumpStage) error
 		if err != nil {
 			return fmt.Errorf("failed to store state jump stage: %w", err)
 		}
+
+		err = bc.dao.Store.Put(storage.SYSCleanStorage.Bytes(), []byte{1})
```

Comment:
Couldn't we put empty byte-slice? We don't use its value anywhere.

Reply:
Fixed!